lib/game.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+require_relative 'board.rb'
+require_relative 'player.rb'
+
+class Game
+  def initialize(player_one, player_two, board)
+    @player_one = player_one
+    @player_two = player_two
+    @board = board
+  end
+
+  def switch_player(active_player)
+    active_player == @player_one ? @player_two : @player_one
+  end
+
+  def check_rows(game_over)
```

Comment:
**[REQUIRED]**
Since all these checks are returning either true or false, you don' need to pass the game_over variable here. Remove the game_over param. And since there are no params, you can also remove the parenthesis.